Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift

```diff
@@ -244,6 +249,15 @@ class SwiftyLineProcessor {
         return foundAttributes
     }
     
+    func processOrderedListRegex(_ text: String) -> String {
+        let regex = try? NSRegularExpression(pattern: "^[0-9]+. ", options: .caseInsensitive)
+        let range = NSMakeRange(0, text.count)
```

Comment:
In other places the pattern has been `NSRange(location: 0, length: string.utf16.count)`

Reply:
Nice catch, updated 👍